theme/style.less

```diff
@@ -73,6 +73,10 @@
     color: @white;
 }
 
+.header-actions .ui.button:not(.red) {
+    color: @black!important;
```

Comment:
would you still need the !important if you overrode it in the pxt-arcade modal.overrides, since it's set in the modal.overrides of pxt?

Reply:
how do overrides work?

Reply:
i think they just get bundled in when the less files build but i'm wondering if you set the rule with the exact same specificity, maybe it would be added after the pxt css and just automatically override

Reply:
Switched! That worked. I hope that this is a good contract to rely on (always comes second in the order). My guess is that it's pretty stable.